docs/integrations/containers-orchestration/opentelemetry/vmware-opentelemetry.md

```diff
@@ -0,0 +1,254 @@
+---
+id: vmware-opentelemetry
```

Comment:
You need to add this article to the left navigation menu. For directions, see "[Navigation menus](https://help.sumologic.com/docs/contributing/style-guide/#navigation-menus)" in our style guid.